src/clj/puppetlabs/trapperkeeper/services/watcher/filesystem_watch_service.clj

```diff
@@ -12,7 +12,7 @@
 
   (init
     [this context]
-    {:watchers (atom [])})
+    (assoc context :watchers (atom [])))
```

Comment:
(x-posting from chat for posterity ...)

`context` here is the service-specific context, not the global TK context for all services.  Thus, for any service, it is always `{}` at the beginning of the `init` function.  So, this change has no effect.